src/python/tests/conftest.py

```diff
@@ -23,17 +23,30 @@
 
 import pytest
 
+test_data_dir_path = Path(__file__).parent
+root_dir_path = test_data_dir_path.parents[2]
+files_dir_path = test_data_dir_path / "flatfiles"
```

Comment:
```suggestion
TEST_DATA_DIR_PATH = Path(__file__).parent
ROOT_DIR_PATH = TEST_DATA_DIR_PATH.parents[2]
FILES_DIR_PATH = TEST_DATA_DIR_PATH / "flatfiles"
```
We could even consider making the constants "private", although not sure how relevant that may be.